### ISSUE TITLE
shoe shop should rank above branded shoe shops

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/iD#10039]
### URL

https://www.openstreetmap.org/edit?editor=id&note=3613014#map=19/50.01805/22.67875

### How to reproduce the issue?

1. create point
2. search for "shoes" in object type

note that some areas (or in future) may have far more brands being listed

### Screenshot(s) or anything else?

![screen16](https://github.com/openstreetmap/iD/assets/899988/307b2a23-ddaa-4a3e-9a4e-51ee2c687d2a)

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

2.27.3

### Which browsers are you seeing this problem on?

Firefox

**Comment by imsidkg** [comment]
Hello @matkoniecz I am new in Open source and the codebase of iD is pretty overwhelming, it would be of great help if you could point me to the files in the repository to be worked on to resolve this issue.

**Comment by matkoniecz** [comment]
I am not familiar with iD codebase.

**Comment by vishwajeetfr** [comment]
I would like to take this issue.
@matkoniecz can you assign it to me

**Comment by matkoniecz** [comment]
@vishwajeetfr You can just start working on it and submit PR

**Comment by tyrasd** [comment]
It would probably make most sense to only include NSI brand presets in the search results when the search result matches the actual _name_ (and _aliases_, _brand_, etc.) of the brand. Then they would not be present for a search of a generic preset like `shoes`.

**Comment by matkoniecz** [comment]
"Office Shoes" brand would still rank above shoe store generic entry.

**Comment by tyrasd** [comment]
Yes, because the actual _name_ of the brand matches the search input.
